spec/helpers_spec.rb

```diff
@@ -0,0 +1,38 @@
+describe "Helpers" do 
+
+  describe 'Helpers#current_user' do 
+    it "returns the current user" do
+      @user1 = User.create(:username => "skittles123", :password => "iluvskittles", :balance => 1000)
+
+      session = {
```

Comment:
Do you need all of this session_id and csrf stuff? That's usually specific to that specific session and shouldn't be added in the tests